ethereum/erc20/hourly_balance_changes.sql

```diff
@@ -0,0 +1,11 @@
+CREATE TABLE IF NOT EXISTS dex.hourly_balance_changes (
```

Comment:
Fair point. I wanted to keep the door open for adding other tokens that are do not conform to the ERC20 token standard. The main goal is indeed caching, but I can imagine also direct use cases such as displaying balance changes. To avoid confusion for now, I changed the name to `weth_hourly_balance_changes`